doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -223,61 +270,33 @@ To prepare the nRF54H20 DK for first use, you must manually program the values o
 Programming the nRF54H20 SoC binaries
 =====================================
 
-After programming the BICR, the nRF54H20 SoC requires the provisioning of the nRF54H20 SoC binaries, a bundle ( :file:`nrf54h20_soc_binaries_v0.6.5.zip`) containing the precompiled firmware for the Secure Domain and System Controller.
+After programming the BICR, the nRF54H20 SoC requires the provisioning of the nRF54H20 SoC binaries, a bundle containing the precompiled firmware for the Secure Domain and System Controller.
 To program the nRF54H20 SoC binaries to the nRF54H20 DK, do the following:
 
-1. Download the `nRF54H20 SoC binaries v0.6.5`_.
+1. Download the nRF54H20 SoC binaries v0.6.8:
+
+   * `nRF54H20 SoC Binaries v0.6.8 for EngC DKs`_, compatible with the nRF54H20 DK v0.8.3 and later revisions
+   * `nRF54H20 SoC Binaries v0.6.8 for EngB DKs`_, compatible with the nRF54H20 DKs ranging from v0.8.0 to v0.8.2.
 
    .. note::
       On MacOS, ensure that the ZIP file is not unpacked automatically upon download.
 
 #. Move the :file:`.zip` bundle to a folder of your choice, then run nRF Util to program the binaries using the following command::
 
-      nrfutil device x-provision-nrf54h --firmware <path-to_bundle_zip_file> --serial-number <serial_number>
+      nrfutil device x-provision-nrf54h --firmware <path-to_bundle_zip_file> --serial-number <serial_number> --traits jlink
 
 .. rst-class:: numbered-step
 
 Transitioning the nRF54H20 SoC to RoT
 =====================================
 
-The current nRF54H20 DK is delivered with its lifecycle state (LCS) set to ``EMPTY``.
-To correctly operate, its lifecycle state must be transitioned to Root of Trust (``RoT``).
-
-.. note::
-   The forward transition to LCS ``RoT`` is permanent.
-   After the transition, it is not possible to transition backward to LCS ``EMPTY``.
-
-To transition the LCS to ``RoT``, do the following:
-
-1. Verify the current lifecycle state of the nRF54H20::
+To transition the nRF54H20 SoC to RoT, do the following:
 
-      nrfutil device x-adac-discovery --serial-number <serial_number>
+#. Transition to the RoT lifecycle state::
 
-   The output will look similar to the following::
+      nrfutil device x-adac-lcs-change --life-cycle rot --serial-number <serial_number> --traits jlink
```

Comment:
`--traits jlink` is redundant since the device is selected using `--serial-number`